src/controls.md

```diff
@@ -10,65 +10,62 @@ same collection located at <https://github.com/adevinta/vulcan-checks>.
-| [vulcan-http-headers](https://github.com/adevinta/vulcan-checks/tree/master/cmd/vulcan-http-headers)                 | Analyzes the HTTP headers using Mozilla Observatory.                                    | DAST - Application                  |
-| [vulcan-ipv6](https://github.com/adevinta/vulcan-checks/tree/master/cmd/vulcan-ipv6)                                 | Checks for IPv6 presence.                                                               | Recon                               |
-| [vulcan-masscan](https://github.com/adevinta/vulcan-checks/tree/master/cmd/vulcan-masscan)                           | Checks if a host has any open port by scanning the whole TCP port range using masscan.  | Recon                               |
-| [vulcan-mx](https://github.com/adevinta/vulcan-checks/tree/master/cmd/vulcan-mx)                                     | Checks for MX DNS Records.                                                              | Recon                               |
-| [vulcan-nessus](https://github.com/adevinta/vulcan-checks/tree/master/cmd/vulcan-nessus)                             | Runs a Nessus scan.                                                                     | DAST - Application & Infrastructure |
-| [vulcan-prowler](https://github.com/adevinta/vulcan-checks/tree/master/cmd/vulcan-prowler)                           | Runs Prowler against an AWS account.                                                    | DAST - Infrastructure               |
-| [vulcan-smtp-open-relay](https://github.com/adevinta/vulcan-checks/tree/master/cmd/vulcan-smtp-open-relay)           | Check for testing SMTP Open Relay.                                                      | DAST - Infrastructure               |
-| [vulcan-spf](https://github.com/adevinta/vulcan-checks/tree/master/cmd/vulcan-spf)                                   | Checks if a domain has a SPF record on DNS.                                             | DAST - Infrastructure               |
-| [vulcan-tenable](https://github.com/adevinta/vulcan-checks/tree/master/cmd/vulcan-tenable)                           | Retrieves findings from assets scanned by Tenable.io.                                   | Finding Collection                  |
-| [vulcan-vulners](https://github.com/adevinta/vulcan-checks/tree/master/cmd/vulcan-vulners)                           | Runs the nmap-script vulners against the target opened ports.                           | DAST - Infrastructure               |
-| [vulcan-wpscan](https://github.com/adevinta/vulcan-checks/tree/master/cmd/vulcan-wpscan)                             | Runs Wordpress scan.                                                                    | DAST - Application                  |
+| Checktype                                                                                                            | Description                                                                             | Security Control   |
```

Comment:
Use the same case as in the first table (i.e. `Data analysis`, `Finding collection`, `Secret detection`).